pkg/services/secrets/manager/manager_test.go

```diff
@@ -280,18 +280,12 @@ func TestSecretsService_Run(t *testing.T) {
 		require.NoError(t, err)
 
 		// Data encryption key cache should contain one element
-		require.Len(t, svc.dataKeyCache, 1)
-
-		// Execute background process after key's TTL, to force
-		// clean up process, during a hundred milliseconds with
-		// gc ticker configured on every nanosecond, to ensure
-		// the ticker is triggered.
-		gcInterval = time.Nanosecond
+		require.Len(t, svc.dataKeyCache.entries, 1)
 
 		t.Cleanup(func() { now = time.Now })
-		now = func() time.Time { return time.Now().Add(dekTTL) }
+		now = func() time.Time { return time.Now().Add(cacheTTLByDefault) }
 
-		ctx, cancel := context.WithTimeout(ctx, 100*time.Millisecond)
+		ctx, cancel := context.WithTimeout(ctx, 1*time.Second)
```

Comment:
Apparently the previous value (100ms) made that test a bit flaky.
So, I think x10ing the value should be enough (🤞🏻 ) and not a big deal.

I'm also open to other suggestions, but I cannot see an easy way to test that feature without big code changes just for the sake of testing.